pmm/pmm2-upgrade-tests.groovy

```diff
@@ -349,12 +349,18 @@ pipeline {
             sh '''
                 ./node_modules/.bin/mochawesome-merge tests/output/parallel_chunk*/*.json > tests/output/combine_results.json || true
                 ./node_modules/.bin/marge tests/output/combine_results.json --reportDir tests/output/ --inline --cdn --charts || true
+                echo --- pmm-managed logs from pmm-server --- >> pmm-managed-full.log
+                docker exec pmm-server cat /srv/logs/pmm-managed.log > pmm-managed-full.log || true
```

Comment:
You will rewrite `pmm-managed-full.log` logs here and the string above will be lost. (we need have >> here)

Reply:
yes, you are right, I just copy pasted wrong. will fix it.